src/sentry/utils/performance_issues/detectors/consecutive_http_detector.py

```diff
@@ -174,3 +174,57 @@ def is_creation_allowed_for_organization(self, organization: Organization) -> bo
 
     def is_creation_allowed_for_project(self, project: Project) -> bool:
         return self.settings["detection_enabled"]
+
+
+class ConsecutiveHTTPSpanDetectorExtended(ConsecutiveHTTPSpanDetector):
+    """
+    Detector goals:
+    - Extend Consecutive HTTP Span Detector to mimic detection using thresholds from
+    - Consecutive DB Queries Detector.
+    """
+
+    type = DetectorType.CONSECUTIVE_HTTP_OP_EXTENDED
+    settings_key = DetectorType.CONSECUTIVE_HTTP_OP_EXTENDED
+
+    def _validate_and_store_performance_problem(self):
+        exceeds_count_threshold = len(self.consecutive_http_spans) >= self.settings.get(
+            "consecutive_count_threshold"
+        )
+
+        exceeds_min_time_saved_duration = False
+        if self.consecutive_http_spans:
+            exceeds_min_time_saved_duration = self._calculate_time_saved() >= self.settings.get(
+                "min_time_saved"
+            )
+
+        exceeds_duration_between_spans_threshold = all(
```

Comment:
This should probably be `does_not_exceed...` or something similar (getting pretty wordy), since it's checking that the span durations are less than the threshold.